test/fixtures/dependencies.js

```diff
@@ -2,7 +2,7 @@ const fs = require('fs');
 const path = require('path');
 const android = require('./android');
 
-const pjson = fs.readFileSync(path.join(__dirname, './files/dependency.json'));
+const pjson = fs.readFileSync(path.join(__dirname, 'files', 'package.json'));
```

Comment:
I'd like not to rely on dependencies we have now. If we need to include something - mock it. Trust me, that will help us keep a code base consistent.